packages/legend-art/src/testMocks/MonacoEditorMockUtils.tsx

```diff
@@ -86,6 +87,7 @@ export const MockedMonacoEditorAPI = {
   setModelLanguage: createMock(),
   defineTheme: createMock(),
   EndOfLinePreference: MockedMonacoEditorEndOfLinePreference,
+  removeAllMarkers: createMock(),
```

Comment:
why are we changing this, is this problem with rebasing?

Reply:
I was trying to monaco in the test so i needed these functions